lib/order.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'script'
+require_relative 'menu'
+require_relative 'dish_lister'
+require_relative 'twilio_adapter'
+
+class Order
+  include DishLister
+
+  attr_reader :menu, :dishes, :complete, :balance
+  def initialize(menu, twilio_adapter, customer_number)
+    @twilio_adapter = twilio_adapter
+    @customer_number = customer_number
+    @menu = menu
+    @dishes = []
+    @balance = 0
+    @complete = false
+  end
+
+  def add_items(*args)
+    order_closed_error
+    new_dishes = args.map { |item| @menu.pick(item) }
+    add_dishes(new_dishes)
+    get_balance
+  end
+
+  def total
+    print_money(@balance)
+  end
```

Comment:
supposedly using the attr_reader is preferred over the instance variable once it's defined.
e.g.
`print_money(balance)` rather than `print_money(@balance)`